### PR TITLE
Revert lost tests

### PR DESCRIPTION
**Pull request by mdrachuk**
Some of the tests were left behind during relocation of `time` to a new repository.

This PR reverts the removed tests and additionally creates a new `testutil-time` module. 
This module holds the utility test services that no-longer belong to `testutil-base`.

### CODE REVIEW

**Comment by mdrachuk** [comment]
@alexander-yevsyukov PTAL.

File: testutil-time/src/main/java/io/spine/test/TimeTests.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        }
+
+        /**
+         * Obtains timestamp in the future a number of seconds from current time.
+         *
+         * @param seconds a positive number of seconds
+         * @return the moment which is {@code seconds} from now
+         */
+        public static Timestamp secondsFromNow(int seconds) {
+            checkPositive(seconds);
```
Comment (alexander-yevsyukov) [inline testutil-time/src/main/java/io/spine/test/TimeTests.java]: Please cover this method with tests.

**Review by alexander-yevsyukov — CHANGES_REQUESTED** [review]
Please see my comment as to the tests.

Consider installing Codecov Chrome plug-in. It would help seeing uncovered code when reviewing PRs in GitHub.

**Review by alexander-yevsyukov — APPROVED** [review]
LGTM

**Pull request merged**